### ISSUE TITLE
Audio Pitches Down/Up after opening an AUP3 project.

### ISSUE THREAD

**Issue by chinakov** [audacity/audacity#6611]
### Bug description

If the User has created a project with multiple audio clips, saves the project with an older version of Audacity, and then re-opens the project in the newest (3.5.1) version of Audacity - some of the clips may be pitched down/up for no reason at all. 

### Steps to reproduce

1. Launch Audacity
2. Open an old project, created with an earlier version of Audacity with Audacity 3.5.1
3. Observe some audio clips/tracks are pitched up/down.

### Expected behavior

The clips should never be pitched up/down without user input. 

### Actual behavior

The clips are pitched/shifted.

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows, macOS, Linux

### Additional context

This is a placeholder ticket that I am fleshing out as I go, as I still can't find the reason for this to happen.

**Comment by saintmatthieu** [comment]
I have followed the steps using Audacity 3.4.2 and couldn't reproduce.

I was then given a project by @chinakov . The problem is that the action leading to the ill state cannot be reproduced anymore. It seems that the newer Audacity auto-corrects settings (in this case wrongly) and auto-saves. I therefore cannot use this project anymore to reproduce the bug.

I observe, though, that all tracks of the project have sample rate 48kHz. After a few experiments, I found out that one of the tracks should actually have had sample rate 44.1kHz, and the others, 192kHz. So I looked for places in the code where 48000 is used as default but found none.

I tried to reproduce the problem following the steps, creating a 3.1.2 project, but didn't succeed. If anyone could provide more specific steps that would be very helpful. I'm otherwise out of ideas.
